embed.html

```diff
@@ -0,0 +1,19 @@
+<!-- Copyright 2019-2022 ETH Zurich and the DaCe authors. All rights reserved. -->
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="./external_lib/material/material-icons.css">
+    <link rel="stylesheet" type="text/css" href="./sdfv.css">
+
+    <script defer src="./dist/sdfv.js"></script>
```

Comment:
Do things like export-to-pdf etc. work without including the external_lib javascript files for that? (blobstream, canvas2pdf, and pdfkit)

Reply:
They don't, and the menu item doesn't appear either. So it's only export to PNG and save SDFG. That should be enough though, as in most embeds you won't even have the menu.